modules/nodes-cluster-resource-override-deploy-cli.adoc

```diff
@@ -0,0 +1,171 @@
+// Module included in the following assemblies:
+//
+// * nodes/clusters/nodes-cluster-overcommit.adoc
+
+[id="nodes-cluster-resource-override-deploy-cli_{context}"]
+= Install the Cluster Resource Override Operator using the CLI
+
+You can use the {product-title} CLI to install the Cluster Resource Override Operator to help control overcommit in your cluster. 
+
+.Prerequisites
+
+* Specify resource limits on containers. Create a LimitRange object with default limits per individual project 
+or configure limits in Pod specs in order for the Cluster Resource Override Operator overrides to apply.
+
+.Procedure
+
+To install the Cluster Resource Override Operator using the CLI:
+
+. Create a Namespace for the Cluster Resource Override Operator:
+
+.. Create a Namespace object YAML file (for example, cro-namespace.yaml) for the Cluster Resource Override Operator:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: clusterresourceoverride-operator
+----
+
+.. Create the Namespace:
++
+----
+$ oc create -f <file-name>.yaml
+----
++
+For example:
++
+----
+$ oc create -f cro-namespace.yaml
+----
+
+. Create a Catalog Source:
+
+.. Create a CatalogSource object YAML file (for example, cro-cs.yaml) for the Cluster Resource Override Operator:
++
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: clusterresourceoverride-catalog
+  namespace: clusterresourceoverride-operator
+  labels:
+    opsrc-provider: custom
+spec:
+  sourceType: grpc
+  image: quay.io/redhat/clusterresourceoverride:latest <1>
+----
+<1> Specify your new Operator catalog image.
+
+.. Create the CatalogSource:
++
+----
+$ oc create -f <file-name>.yaml
+----
++
+For example:
++
+----
+$ oc create -f cro-cs.yaml
+----
+
+. Wait for Ready state:
++
+----
+$ watch oc -n clusterresourceoverride-operator get catalogsource clusterresourceoverride-catalog -o jsonpath='{.status.connectionState.lastObservedState}'
+----
+
+. Create an Operator Group:
+
+.. Create an OperatorGroup object YAML file (for example, cro-og.yaml) for the Cluster Resource Override Operator:
++
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-operators-redhat
+  namespace: openshift-operators-redhat 
+spec: {}
+----
+
+.. Create the Operator Group:
++
+----
+$ oc create -f <file-name>.yaml
+----
++
+For example:
++
+----
+$ oc create -f cro-og.yaml
+----
+
+. Create a Subscription:
+
+.. Create an Subscription object YAML file (for example, cro-sub.yaml) for the Cluster Resource Override Operator:
++
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: clusterresourceoverride
+  namespace: clusterresourceoverride-operator
+spec:
+  channel: "4.4"
+  name: clusterresourceoverride
+  source: clusterresourceoverride-catalog
+  sourceNamespace: clusterresourceoverride-operator
```

Comment:
Since our operator is available as a RedHat operator in OperatorHub, we will use the following
```
    source: redhat-operators
    sourceNamespace: openshift-marketplace
```

If you have access to a 4.4 rc cluster where the operator is available you can double check the value by installing the operator and looking at the Subscription object created by OperatorHub.

Reply:
Fixed